cmd/frpc/sub/http.go

```diff
@@ -46,6 +46,7 @@ func init() {
 	httpCmd.PersistentFlags().StringVarP(&hostHeaderRewrite, "host_header_rewrite", "", "", "host header rewrite")
 	httpCmd.PersistentFlags().BoolVarP(&useEncryption, "ue", "", false, "use encryption")
 	httpCmd.PersistentFlags().BoolVarP(&useCompression, "uc", "", false, "use compression")
+	httpCmd.PersistentFlags().BoolVarP(&tlsEnable, "tls_enable", "", false, "enable frpc tls")
```

Comment:
You can improve this flag register section after this PR has been merged. 

It's not convenient to add new flag since we should add it to all different commands.

It's better to add a new function like `RegisterCommonFlags(cmd *cobra.Common)` and call it for each command.

Reply:
got it